jgrapht-core/src/main/java/org/jgrapht/graph/DirectedAcyclicGraph.java

```diff
@@ -15,71 +15,61 @@
  * (b) the terms of the Eclipse Public License v1.0 as published by
  * the Eclipse Foundation.
  */
-package org.jgrapht.experimental.dag;
+package org.jgrapht.graph;
 
 import java.io.*;
 import java.util.*;
 
 import org.jgrapht.*;
-import org.jgrapht.graph.*;
 import org.jgrapht.traverse.*;
 
 /**
+ * A directed acyclic graph (DAG).
+ * 
  * <p>
- * DirectedAcyclicGraph implements a DAG that can be modified (vertices &amp; edges added and
- * removed), is guaranteed to remain acyclic, and provides fast topological order iteration.
- * </p>
+ * Implements a DAG that can be modified (vertices &amp; edges added and removed), is guaranteed to
```

Comment:
The two main advantages of this class are:
1. fast topo order iteration
2. guaranteed acyclic. 

Unfortunatley, (2) comes with a hefty cost: each arc addition no longer runs in constant time, but seems to run somewhere in the order of the number of vertices in the graph.
So the end user is now facing the following choice:
1. use `SimpleDirectedGraph` with `TopologicalOrderIterator`
2. use the `DirectedAcyclicGraph` class.
Correct me if I'm wrong, but it doesn't seem to be clear upfront which of these 2 approaches is more efficient?
Any particular use cases where this class comes in handy? It seems that in many cases choice (1) is preferred, especially when the graph is large and the TopologicalOrderIterator isn't invoked often. Can we state something about the runtime complexities in the class description?